# HG changeset patch
# User Michael Busby <mibusby@gmail.com>
# Date 1360316996 28800
# Node ID 072857158e48876777a595f4d25d3a6b1dbda905
# Parent  78db16ac10e5d3f9244dda3cf05c2465121cd148
Add the tf2 build job only if the ep2v sdk is available

diff -r 78db16ac10e5 -r 072857158e48 AMBuildScript
--- a/AMBuildScript	Fri Feb 08 22:46:49 2013 -0500
+++ b/AMBuildScript	Fri Feb 08 01:49:56 2013 -0800
@@ -425,8 +425,12 @@
 		['extensions', 'topmenus', 'AMBuilder'],
 		['extensions', 'updater', 'AMBuilder'],
 		['extensions', 'sqlite', 'AMBuilder'],
-		['extensions', 'regex', 'AMBuilder'],
-		['extensions', 'tf2', 'AMBuilder'],
+		['extensions', 'regex', 'AMBuilder']
+	]
+if 'ep2v' in sm.sdkInfo:
+	FileList.append(['extensions', 'tf2', 'AMBuilder'])
+
+FileList += [
 		['sourcepawn', 'jit', 'AMBuilder'],
 		['sourcepawn', 'compiler', 'AMBuilder'],
 		['plugins', 'AMBuilder'],
